search_service/__init__.py

```diff
@@ -81,17 +82,20 @@ def create_app(*, config_module_class: str) -> Flask:
     api_bp.add_url_rule('/healthcheck', 'healthcheck', healthcheck)
     api = Api(api_bp)
     # Table Search API
-    # TODO: Rename endpoint to be more generic and accept a resource type so that logic can be re-used
+
     api.add_resource(SearchTableFilterAPI, '/search_table')
+    # TODO: Rename endpoint to be more generic and accept a resource type so that logic can be re-used
```

Comment:
Will you be doing this later as part of this ticket? We could always decide as a team whether we do the backwards incompatible change now, or hold it for a planned major release.

Reply:
yeah, I actually change my mind to prefer to change it once this endpoint is in as changing the endpoint includes two variables.